iris-mpc-cpu/src/hnsw/searcher.rs

```diff
@@ -371,16 +371,17 @@ impl HnswSearcher {
         let eval_dist_span = trace_span!(target: "searcher::cpu_time", "eval_distance_batch_aggr");
         let less_than_span = trace_span!(target: "searcher::cpu_time", "less_than_aggr");
         let insert_span =
-            trace_span!(target: "searcher::cpu_time", "insert_into_sorted_neighborhood_aggr");
+            trace_span!(target: "searcher::cpu_time", "insert_into_sorted_NeighborhoodV<V>_aggr");
 
         // Continue until all current entries in candidate nearest neighbors list have
         // been opened
-        while let Some(c) = W
-            .edges
-            .iter()
-            .map(|(c, _)| c)
-            .find(|&c| !opened.contains(c))
-        {
+        loop {
+            let c = W.iter().map(|(c, _)| c).find(|&c| !opened.contains(c));
+            if c.is_none() {
+                break;
+            }
+            let c = c.unwrap();
```

Comment:
Please, leave a comment that this `unwrap` guarantees to succeed. It is needed for code audits.